integration/api/src/main/java/org/phenotips/variantStoreIntegration/VariantStoreVariant.java

```diff
@@ -62,17 +63,20 @@ public VariantStoreVariant(GAVariant gaVariant, Integer totIndividuals) {
             setScore(Double.valueOf(value));
         }
 
+        DecimalFormat df = new DecimalFormat("#.####");
```

Comment:
Shouldn't this be created only once as an instance member, instead of once for each variant?